fv3gfs-physics/fv3gfs/physics/stencils/microphysics.py

```diff
@@ -794,7 +808,7 @@ def sedimentation(
 
         with interval(0, 1):
 
-            if tz > constants.tice:
+            if tz > tice:
                 stop_k = 1
```

Comment:
So we cannot have temporary boolean right? Is this something that we should support when we add typed temporaries?

Reply:
we can, it'll just be a 3d boolean